### PR TITLE
Integrate showing transactions & layout of wallet page

### PR DESCRIPTION
**Pull request by buoyad**
The wallet page will be a native `SectionList` eventually. I added a desktop port to common adapters and used it to set up the layout of the wallet page. Also makes some changes to the transaction component to support transactions that don't have a display currency. r? @keybase/react-hackers

### CODE REVIEW

**Comment by buoyad** [comment]
Affected storyshots are for the transactions, assets, and wallet rows

**Comment by buoyad** [comment]
Hm, I can't repro, when I concatenate the returned asset array to itself in the saga I end up with two identical rows in the GUI for it. How did you lengthen the array?

File: shared/wallets/wallet/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    if (section.title === 'Your assets') {
+      children.push(
+        <Asset accountID={props.accountID} index={item.item} key={`${props.accountID}:${item.item}`} />
+      )
+      if (index !== section.data.length - 1) {
+        // don't put divider after last thing in section
+        children.push(<Divider key={`${props.accountID}:${item.item}:divider`} />)
+      }
+    }
+    // TODO
```
Comment (cjb) [inline shared/wallets/wallet/index.js]: (what's the TODO here?)

Comment (buoyad) [reply shared/wallets/wallet/index.js]: to render the transaction rows, I'll remove it in the second PR

**Pull request merged**